### DISCUSSION TITLE
Should we use OpenMage internal notifications to inform about new Releases?

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#2909] [General]
There were notes that we are not using the internal notifications enough, and that maybe we should use it to inform about new Releases.

What kind of Information would you like to get over it? If for Releases, we should keep in mind automation and that most of the time we have 2 Releases in parallel. So how closely should we let it happen? would be the next day acceptable? Or a summary once a Week or Month or Quarter?

Any other details?

**Comment by tmotyl** [comment]
It should be delayed at last few days. To make room for fixing regressions before asking everybody to upgrade

**Comment by addison74** [comment]
The purpose of using this facility should be to transmit essential information to OpenMage users. It should have been used for a long time, since the EOL for Magento 1 was announced, in order not to cause people to reorient themselves.  If we look at the closed PRs from the Web_Notifications repository, we notice that their number is only 3 in almost 4 years, two in 2019 and one recently about the war in Ukraine. On the one hand, Magento transmits that the EOL is coming and OpenMage does not transmit anything, it is more than certain that the reorientations/migrations were also made due to the lack of information about this project.

As essential information I consider the release of new versions, vulnerabilities, requests for opinions  (this is essential to keep in touch with the community, not only with those here). The format can be similar to Magento (severity, title, description, url). In general, those that do not pursue an advertising, material purpose, which annoys the users.

For the **title**, I would keep the format that many are familiar with "_OpenMage version X.X.X.X is now available"_ or _"Important Security Update - Zend Platform Vulnerability"_.

For the **description**, the messages should be _"We are happy to announce the availability of Magento Preview Version CE 1.5.0.0-rc1 for download. We introduced new features like ... and fixed several bugs. Please visit the link for more information."_ or _"We have recently learned of a serious vulnerability in the Zend platform on which Magento is built. Learn more and access a patch that addresses this issue."_

For **links** you can use posts created in the Discussions section (including polls for voting) or the dedicated pages of the new versions.

Regarding the announcements for a new version, it can wait 1-2 weeks, this is enough to get a feedback from those who upgrade immediately. In the case of vulnerabilities, the patches must be published immediately.

I hope that all this will not come too late.

**Reply by fballiano** [reply]
this is a great answer that should be bookmarked for when we'll need to rollout the next notification

**Comment by jonashrem** [comment]
I agree with release notifications after a new version. 

It might also be useful to inform about changes in the requirements. Like "PHP 8.2 is supported now". 

Having regular notifications also should order the risk of people thinking they are hacked when they get one (see the Twitter discussion)

**Comment by fballiano** [comment]
slight delay as @tmotyl was suggesting, 1 week is fine.

we have 2 releases but they need to be in the same notification, kinda the way we do it for the twitter/linkedin/etc announcements.

**Comment by pauldpauld** [comment]
I also agree with release notifications after a new version.
I am probably one of the "smaller" users of OpenMage, one person business, my own server, I stayed with Magento 1.9.X for ages as I looked at Magento2 and it horrified me!
I think many people may be stuck on Magento1 because they don't realise OpenMage is available.

I would like to suggest (for me and possibly others) some link we can place in the footer of our websites linking to OpenMage, I understand that might not suit a lot of people, but I don't have programming skills and I DO see a lot of work done by the contributors to this project, so perhaps this might be my way of helping? 
Perhaps an option in Configuration/Design?

I subscribe to this project and I read all the emails/messages/discussions and  I say a big THANK YOU to all of you who work so hard on this! 

Thanks, Paul

**Comment by addison74** [comment]
If we take a look to this web page https://www.openmage.org/Web_Notifications/index it looks totally unprofessional. The page is generated by Web_Notifications content. Anyone who visits this page immediately comes to the conclusion that nothing important happened over the years. I think the time has come to discuss about OpenMage future, but also to quickly correct all these incomprehensible delays that have negative impact on everyone's work.

It is to be appreciated that this project still has a handful of people who want to make OpenMage a solid alternative to the other available shopping carts. In the last year it was a pleasant surprise to learn new things. I personally consider that when Adobe announced the EOL for Magento 1, not all the necessary efforts were made to continue this project with those interested. The migration to M2, the choice of other platforms such as those based on Wordpress, also come as a result of not being properly informed about what OpenMage wants. Even if the owners of the project did their best they did not convince enough. Meanwhile OpenMage got interesting features and I still consider it a high-performance shopping cart for these times.
